assets/common/components/register_model/spec.yaml

```diff
@@ -9,7 +9,7 @@ display_name: Register model
 description:
   Register a model to a workspace or a registry. The component works on compute with [MSI](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-manage-compute-instance?tabs=python) attached.
 
-environment: azureml://registries/azureml/environments/python-sdk-v2/versions/20
+environment: azureml://registries/azureml-staging/environments/python-sdk-v2/versions/21
```

Comment:
Is this change accidental?

Reply:
Thanks for noticing this Sarthak. Updated now